components/core/src/os/users/mod.rs

```diff
@@ -16,10 +16,10 @@
 mod windows;
 
 #[cfg(windows)]
-pub use self::windows::{get_uid_by_name, get_gid_by_name, get_effective_uid, get_home_for_user};
+pub use self::windows::{get_uid_by_name, get_gid_by_name, get_effective_uid, get_home_for_user, get_current_username, get_current_groupname};
```

Comment:
@mwrock we can move to an `*` on the import and move the `GetUserTokenStatus` to a non-public function.  It's not a big deal, it does compile and doesn't seem to break anything.  I'm not opposed to doing the `pub use` that way since we control that crate.